pyproject.toml

```diff
@@ -22,5 +22,5 @@ classifiers = [
 ]
 
 [tool.meson-python.args]
-setup = ['-Dcpp_std=c++latest']
+setup = ['-Dcpp_std=c++23', '--debug']
```

Comment:
You build the windows wheel with visual studio? Interesting, msys or clang?

latest doesn't work on any linux gcc compiler. Is there a way to specify it without having a windows version and a linux version?

Reply:
No, just plain Visual Studio (dev tools cmd.exe). The deps come from WrapDB. One of the goals of CapyPDF was to examine how easy and/or difficult it would be to build wheels without any unixy tools.

> Is there a way to specify it without having a windows version and a linux version?

FWICT, no, because toml does not provide for it. Hence me needing to add c++23 to VS.

Reply:
>  without any unixy tools.

Ah open sourcing on nightmare mode. :wink: 

Does the additional instruction to add C++ to VS need to be added to the readme or contributor documentation?

Reply:
> Ah open sourcing on nightmare mode

Actually it is 10x simpler than trying to use Unixy package managers. Even if some of your dep is not is in WrapDB it's typically a lot less effort to port and submit it. 

> Does the additional instruction to add C++ to VS need to be added to the readme or contributor documentation?

I'd prefer to fix this upstream. Hopefully tomorrow. CapyPDF requires users to have latest Meson from master anyway for now.